src/main/java/org/bytedeco/javacpp/tools/Context.java

```diff
@@ -53,6 +53,7 @@ class Context {
         templateMap = c.templateMap;
         usingList = c.usingList;
         namespaceMap = c.namespaceMap;
+        explicitUpcast = c.explicitUpcast;
```

Comment:
By definition, "casting" is always *explicit*, so please strip out that term from everywhere.

Reply:
What about "static" instead of "explicit" ? Cast alone is used elsewhere else will add confusion for the code reader.

Reply:
"upcast" isn't used anywhere else.